roles/edpm_container_quadlet/defaults/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+edpm_container_quadlet_kolla_config_dir: /var/lib/kolla/config_files
+edpm_container_quadlet_systemd_config_dir: "/etc/containers/systemd/"
+edpm_container_quadlet_service: ""
+edpm_container_quadlet_service_volumes: []
+edpm_container_quadlet_service_use_host_network: false
+edpm_container_quadlet_service_image: ""
+# List of common volumes that can be included in the template of a container
+# definition to include the most common volumes.
+edpm_container_quadlet_common_volumes:
+  - path: /etc/hosts
+    mountPath: /etc/hosts
+    readOnly: true
+    name: etc-hosts
+  - path: /etc/localtime
+    mountPath: /etc/localtime
+    readOnly: true
+    name: etc-localtime
+  # OpenSSL trusted CAs
+  - path: /etc/pki/ca-trust/extracted
+    mountPath: /etc/pki/ca-trust/extracted
+    readOnly: true
+    name: ca-extracted
+  - path: /etc/pki/ca-trust/source/anchors
+    mountPath: /etc/pki/ca-trust/source/anchors
+    readOnly: true
+    name: anchors
+  - path: /etc/pki/tls/certs/ca-bundle.crt
+    mountPath: /etc/pki/tls/certs/ca-bundle.crt
+    readOnly: true
+    name: ca-bundle
+  - path: /etc/pki/tls/certs/ca-bundle.trust.crt
+    mountPath: /etc/pki/tls/certs/ca-bundle.trust.crt
+    readOnly: true
+    name: ca-trust-bundle
+  - path: /etc/pki/tls/cert.pem
+    mountPath: /etc/pki/tls/cert.pem
+    readOnly: true
+    name: tls-cert
```

Comment:
i assume that all of the above mounts under /etc/pki will always exist?

Reply:
Pretty sure they do, I'll check the base bootc image to be sure.

Reply:
Yeah, these are always there:

```
❯ podman run -it registry.redhat.io/rhel9/rhel-bootc:9.4 ls /etc/pki/tls
cert.pem  certs  ct_log_list.cnf  fips_local.cnf  misc  openssl.cnf  openssl.d  private

❯ podman run -it registry.redhat.io/rhel9/rhel-bootc:9.4 ls /etc/pki/ca-trust
README  ca-legacy.conf  extracted  source

❯ podman run -it registry.redhat.io/rhel9/rhel-bootc:9.4 ls /etc/pki/ca-trust/source
README  anchors  blocklist  ca-bundle.legacy.crt
```

Reply:
CentOS Stream:
```
❯ podman run -it quay.io/centos-bootc/centos-bootc:stream9 ls /etc/pki
ca-trust  consumer  entitlement  entitlement-host  fwupd  fwupd-metadata  java  nssdb  rpm-gpg  tls
```

RHEL:
```
❯ podman run -it registry.redhat.io/rhel9/rhel-bootc:9.4 ls /etc/pki/tls
cert.pem  certs  ct_log_list.cnf  fips_local.cnf  misc  openssl.cnf  openssl.d  private
```

So, yeah, they both have that file available. We should be fine with the default.

Reply:
CentOS Stream:
```
❯ podman run -it quay.io/centos-bootc/centos-bootc:stream9 ls /etc/pki
ca-trust  consumer  entitlement  entitlement-host  fwupd  fwupd-metadata  java  nssdb  rpm-gpg  tls
```

RHEL:
```
❯ podman run -it registry.redhat.io/rhel9/rhel-bootc:9.4 ls /etc/pki/tls
cert.pem  certs  ct_log_list.cnf  fips_local.cnf  misc  openssl.cnf  openssl.d  private
```

So, yeah, they both have that file available. We should be fine with the default.